statistics/cmsketch_test.go

```diff
@@ -34,6 +34,18 @@ func (c *CMSketch) insert(val *types.Datum) error {
 	return nil
 }
 
+func insertTopN(d, w int32, val []*types.Datum, n uint32, total uint64) (*CMSketch, error) {
```

Comment:
Can we define this as a member function of *CMSketch like the `insert` in line 28 does?